src/routes/(blog-article)/how-to-setup-the-development-environment/+page.md

```diff
@@ -2,7 +2,7 @@
 title: How To Setup The Dev Env
 slug: how-to-setup-the-development-environment
 coverImage: /images/posts/development-environment.png
-date: 2023-07-11T12:29:04.295Z
+date: 2024-04-18T12:29:04.295Z
```

Comment:
Hi @ivan9989-pkj, this is the publication date. You have to keep the original. [We want to add the update date in the future](https://github.com/torrust/torrust-website/issues/151).